### PR TITLE
Fix spec errors

### PR DESCRIPTION
**Pull request by dc-mattj**

### CODE REVIEW

**Comment by dc-mattj** [comment]
Workaround bug in rspec with ruby 1.8.7
Fix various indenting issues
Remove non-existent test case
Correct file permissions and ownership tests

**Comment by yankcrime** [comment]
👍🏻

File: spec/classes/radius_spec.rb [hunk 1]
```diff
@@ -59,9 +59,5 @@
       it { should contain_service('radius').with_ensure('stopped') }
     end
 
-    context 'with a custom config_file' do
-      let(:params) { { 'config_file' => '/etc/dave.conf' } }
-      it { should contain_file('/etc/dave.conf')}
-    end
```
Comment (spjmurray) [inline spec/classes/radius_spec.rb]: Is this test no longer valid?  e.g. behaviour is deprecated

Comment (dc-mattj) [inline spec/classes/radius_spec.rb]: It doesn't have an option to use a custom config file

**Pull request merged**